app/views/shared/forms/proceedings_types/_proceeding_type.html.erb

```diff
@@ -8,7 +8,7 @@
   <div class="govuk-grid-column-one-third control-column">
     <p class="govuk-body govuk-!-margin-bottom-1 govuk-!-margin-top-4">
       <%= link_to(
-            t('generic.remove'),
+            t("#{translation_path}.link_title"),
```

Comment:
I'd suggest this as being a little cleaner:
```ruby
t('.link_title', scope: translation_path)
```
See https://guides.rubyonrails.org/i18n.html#looking-up-translations